packages/geany.rb

```diff
@@ -41,12 +46,16 @@ def self.install
   end
 
   def self.postinstall
+    # This is needed to aoid "Error loading theme icon 'geany-build' for stock" messages.
```

Comment:
avoid?  Missing v?  What package provides `gtk-update-icon-cache`?  If it is a dependency then no need to check if the binary exists first.

Reply:
True too. I think gtk3 has that, so just adding the dependency should take care of it. And yes that's a typo.

Reply:
Looks like gtk3 & gtk2 have it, so this is already covered...